.automation/build.py

```diff
@@ -2757,7 +2757,9 @@ def manage_output_variables():
                 updated_versions = 1
                 break
         if updated_versions == 1:
-            print("::set-output name=has_updated_versions::1")
+            output_file = os.environ["GITHUB_OUTPUT"]
```

Comment:
If we are not on github (like when building locally) the variable will be empty, the code has to manage such case ^^ ( if ^^ )

Reply:
What should happen when not running on GitHub? Print to console like it does now?

Reply:
In my opinion, nothing at all. The output environment file and deprecated `set-output` command are GitHub Actions concepts.